docs/user/site-health.md

```diff
@@ -0,0 +1,110 @@
+| User Groups: Group Activity Deletions | Indicates whether group administrators and moderators to delete activity items from their group's activity stream is allowed. The Default is **enabled**. Configurable from the [BuddyPress Options](administration/settings/options.md) screen. | User Groups |
+| Activity Streams: Post Comments | Indicates whether activity stream commenting on posts and comments is allowed. The Default is **disabled**. Configurable from the [BuddyPress Options](administration/settings/options.md#post-comments) screen. ***Note**: The Site Tracking component must be **active** in order to **enable** this feature*. |Activity Streams & Site Tracking |
+| Activity Streams: Activity auto-refresh | Indicates whether a check for new items while viewing the activity stream is automatically allowed. The Default is **enabled**. Configurable from the [BuddyPress Options](administration/settings/options.md#activity-auto-refresh) screen. |Activity Streams |
+
+### (2) BuddyPress Constants
+
+![site health - BuddyPress panel](assets/shs-buddypress-constants-02.png)
+
+The image above is a truncated example of an expanded BuddyPress Constants panel. Below is the full list of BuddyPress Constants and their associated default values.
+
+> [!Note]
+> *A constant is an identifier (name) for a simple value. As the name suggests, that value cannot change during the execution of the script (except for magic constants, which aren't actually constants). Constants are case-sensitive. By convention, constant identifiers are always uppercase*. [PHP Manual - Constants](https://www.php.net/manual/en/language.constants.php)
+
+Some of these Constants are user-definable and will alter the behavior of BuddyPress when the defaults are overridden. This list is made available for your purvey in the event a misdefined Constant may be causing unexpected behavior.
+
+| Constant | Default Value |
+| :------- | :------------ |
+| BP_VERSION | The installed version of BuddyPress. |
+| BP_DB_VERSION | The installed database version of BuddyPress. |
+| BP_REQUIRED_PHP_VERSION | The minimum supported PHP version. |
+| BP_PLUGIN_DIR | The filesystem directory path (with trailing slash) for the BuddyPress plugin. |
+| BP_PLUGIN_URL | The URL directory path (with trailing slash) for the BuddyPress plugin. |
+| BP_IGNORE_DEPRECATED | undefined |
+| BP_LOAD_DEPRECATED | undefined |
+| BP_ROOT_BLOG | 1 (integer) |
+| BP_ENABLE_MULTIBLOG | undefined |
+| BP_ENABLE_ROOT_PROFILES | undefined |
+| BP_DEFAULT_COMPONENT | undefined |
+| BP_XPROFILE_BASE_GROUP_NAME | If **Extended Profiles** component is active this value will be the default avatar setting as selected per the [Settings Discussion](https://wordpress.org/documentation/article/settings-discussion-screen/#avatars) screen. Otherwise, the value is undefined.  |
```

Comment:
```suggestion
| BP_XPROFILE_FULLNAME_FIELD_NAME | If **Extended Profiles** component is active this value will be the primary xprofile field label (see [BuddyPress xProfile fields administration](administration/users/xprofile.md)). Otherwise, the value is undefined. |
```


Changesets [14049](https://buddypress.trac.wordpress.org/changeset/14049) & [14050](https://buddypress.trac.wordpress.org/changeset/14050) changed/corrected the value for `BP_XPROFILE_BASE_GROUP_NAME` which requires this suggested update.
```

Reply:
@emaralive I believe it should be about the `BP_XPROFILE_BASE_GROUP_NAME` constant not about the `BP_XPROFILE_FULLNAME_FIELD_NAME` one which is documented at line 74.

That is to say: 
```md
| BP_XPROFILE_BASE_GROUP_NAME | If **Extended Profiles** component is active this value will be the label of the primary group of xProfile fields (see [BuddyPress xProfile fields administration](administration/users/xprofile.md)). Otherwise, the value is undefined. |
```

Reply:
@imath - Yes, you are correct. I had copied and pasted from line 74 and didn't edit any further due to some unknown reason, probably due to a distraction that occurred around the same time.

The goal was to earmark the places where I would make edits and then push another commit.  That stated,  I updated the suggestion and if there is nothing else, I'll updatte .md with the text changes and replace the indicated screenshot with a newer one and then push a commit either later today or tomorrow.